packages/web/app/components/PatientPrinting/modals/IPSQRCodeModal.jsx

```diff
@@ -29,7 +30,13 @@ export const IPSQRCodeModal = React.memo(({ patient }) => {
 
   return (
     <FormModal
-      title="International Patient Summary"
+      title={
+        <TranslatedText
+          stringId="patient.ips.title"
```

Comment:
```suggestion
          stringId="patient.modal.ps.title"
```
I think thats more consistent